contrib/mesos/pkg/controllermanager/controllermanager.go

```diff
@@ -160,8 +161,8 @@ func (s *CMServer) Run(_ []string) error {
 		routeController.Run(s.NodeSyncPeriod)
 	}
 
-	resourceQuotaController := resourcequotacontroller.NewResourceQuotaController(kubeClient)
-	resourceQuotaController.Run(s.ResourceQuotaSyncPeriod)
+	go resourcequotacontroller.NewResourceQuotaController(
```

Comment:
why does this need to be in a goroutine?  I thought Run was supposed to launch children?